hubmap/frontend/src/middleware/index.js

```diff
@@ -0,0 +1,7 @@
+/** *
```

Comment:
```suggestion
/**
```
Please fix all occurrences, unless there's a reason to do this?